### ISSUE TITLE
bt_interface: "disable" not working in YAML config

### ISSUE THREAD

**Issue by djkakadu** [custom-components/ble_monitor#681]
Hello.
I have a little problem with disabling a bt interface.

If i use these configuration: 

```
ble_monitor:
  bt_interface: "disable"
```

After restart a Home Assistant i found these on my log:

```
2022-01-31 18:47:10 ERROR (MainThread) [custom_components.ble_monitor] Bluetooth interface with MAC address D is not available
2022-01-31 18:47:10 ERROR (MainThread) [custom_components.ble_monitor] Bluetooth interface with MAC address I is not available
2022-01-31 18:47:10 ERROR (MainThread) [custom_components.ble_monitor] Bluetooth interface with MAC address S is not available
2022-01-31 18:47:10 ERROR (MainThread) [custom_components.ble_monitor] Bluetooth interface with MAC address A is not available
2022-01-31 18:47:10 ERROR (MainThread) [custom_components.ble_monitor] Bluetooth interface with MAC address B is not available
2022-01-31 18:47:10 ERROR (MainThread) [custom_components.ble_monitor] Bluetooth interface with MAC address L is not available
2022-01-31 18:47:10 ERROR (MainThread) [custom_components.ble_monitor] Bluetooth interface with MAC address E is not available
2022-01-31 18:47:10 WARNING (MainThread) [custom_components.ble_monitor] No configured Bluetooth interface was found, using default interface instead
```

My configuration.yaml look these:
```
ble_monitor:
    bt_interface: "disable"
    devices:
    - mac: 'A4:C1:38:FC:12:F6'
      name: "Spálen"
    - mac: 'A4:C1:38:40:4F:55'
      name: "Obývačka"
    - mac: 'E9:24:90:5D:71:E0'
      name: "Katka Hodinky"
      track_device: True
    - mac: '70:87:9E:92:A4:2F'
      name: "Mi Scale Váha"
```
I need disable bt and use esphome parsing mode, but i have a problem , please help :)

**Comment by Ernst79** [comment]
Will look into it, it probably work if you use configuration via the UI. I havent checked YAML mode, but there seems to be a bug, I agree.

**Comment by djkakadu** [comment]
> Will look into it, it probably work if you use configuration via the UI. I havent checked YAML mode, but there seems to be a bug, I agree.

When i delete Yaml config and switch to UI config, everything work. Thanks for reply.

**Comment by Ernst79** [comment]
Fixed in 7.4.1-beta

**Comment by Ernst79** [comment]
Fixed in 7.5.0
